css/cart.css

```diff
@@ -0,0 +1,44 @@
+.cart-main {
+  max-width: 1200px;
+  margin: 1rem auto;
+}
+
+.cart-details {
+  flex-basis: 60%;
+}
+
+.price-details {
+  flex-basis: 40%;
+  height: fit-content;
+}
+
+.cart-action-btn {
+  display: flex;
+  justify-content: flex-end;
+  box-shadow: 0px 0px 4px var(--gray);
+  padding: 1rem 0.5rem;
+}
+
+.cart-action-btn .btn {
+  padding: 0.8rem 3rem;
+}
+
+.product-quantity-count {
+  width: 2.5rem;
+  height: 0.875rem;
+}
+
+.complementary-ouline-btn {
+  outline: 1px solid var(--complementary-color) !important;
+}
+
+.complementary-ouline-btn:hover {
+  background-color: var(--complementary-color);
+  color: var(--white);
+}
+
+@media screen and (max-width: 769px) {
```

Comment:
As it is particularly targeting iPads and mobiles, you can use the more common breakpoint of `768px`

Reply:
> Good work done, have suggested changes that can make the code better, query regarding use of `!important` is mentioned
> 
> Additionally, I would also suggest to place the `Place Order` button below the price details table, instead of where it currently is placed (bottom of page), so that it is visible to the user and fulfills it's purpose of being a Call To Action button.

Place order button moved to the price detail section